src/main/scala/com/raquo/laminar/keys/CompositeKey.scala

```diff
@@ -0,0 +1,161 @@
+package com.raquo.laminar.keys
+
+import com.raquo.airstream.core.Observable
+import com.raquo.laminar.api.Laminar.{MapValueMapper, StringValueMapper}
+import com.raquo.laminar.keys.CompositeKey.CompositeValueMapper
+import com.raquo.laminar.modifiers.{Binder, Setter}
+import com.raquo.laminar.nodes.ReactiveElement
+
+import scala.scalajs.js
+
+// @TODO[Performance] We can eventually use classList for className attribute instead of splitting strings. That needs IE 10+
+// @TODO[Performance] Will string splitting be faster with native JS method? How do we access it?
```

Comment:
I think

```
import scala.scalajs.js.JSStringOps._
```

puts `<String>.jsSplit(...)` extension method in your scope: https://github.com/scala-js/scala-js/blob/master/library/src/main/scala/scala/scalajs/js/JSStringOps.scala#L162

Reply:
Nice, thanks!